src/app/mosh_client.js

```diff
@@ -214,6 +217,7 @@ function updateMode(e) {
     keyRow.hidden = true;
     remoteCommandRow.hidden = false;
     serverCommandRow.hidden = false;
+    moshEscapeKeyRow.hidden = false;
```

Comment:
Since this is `false` in both cases, please omit this line and fixup the HTML to be enabled by default.